syntax = "proto3";

package compiler;

service OddsCompilerService {
    // Returns a Over/Under goals odds for a given fixture
    rpc GetOverUnderGoalsForFixture(OverUnderRequest) returns (OverUnderGoalsResponse) {}
}

message OverUnderRequest {
    uint64 fixture_id = 1;
}

message OverUnderGoalsResponse {
    uint64 fixture_id = 1;
    float under = 2;
    float over = 3;
}

